warehouse/models/mart/gtfs_quality/fct_monthly_reports_site_organization_guideline_checks.sql

```diff
@@ -0,0 +1,54 @@
+{{ config(materialized='table') }}
+
+WITH idx_monthly_reports_site AS (
+    SELECT *
+    FROM {{ ref('idx_monthly_reports_site') }}
+),
+
+generate_biweekly_dates AS (
+    SELECT DISTINCT
+        publish_date,
+        sample_dates
+    FROM idx_monthly_reports_site
+    LEFT JOIN
+        UNNEST(GENERATE_DATE_ARRAY(
+            -- add a few days because otherwise it will always pick the first of the month,
+            -- which may be disproportionately likely to have new feed published and thus issues
+            DATE_ADD(CAST(date_start AS DATE), INTERVAL 3 DAY),
+            CAST(date_end AS DATE), INTERVAL 2 WEEK)) AS sample_dates
+),
+
+checks AS (
```

Comment:
For stylistic consistency I would move this above the prior CTE because it has a `ref`